content/asp.net/getting-started/controllers-actions.md

```diff
@@ -1,20 +1,122 @@
 
-Start the lesson here.
+Actions are responsible for ensuring that incoming requests are valid and that appropriate responses are returned to the client, typically building up and sending a model to a [view](views.md) for rendering. You can put whatever logic is required in an action to respond accordingly to requests, but the purpose of controllers and actions is to help separate view rendering from business logic, not to provide a place to put your business logic. Business logic should be located in services that are passed into your controller using [dependency injection](controller-dependencies.md). 
 
-...
+Each action has a role in constructing a proper HTTP response to the client, but you don't have to go it alone; the Framework has a number of features that will help you do so. You can start to  explore those features by creating a new class in the controllers folder called `PersonController`. Update the class so that it appears as below:
 
+```c#
+using Microsoft.AspNetCore.Mvc;
+
+namespace MvcBasics.Controllers
+{
+    public class PersonController : Controller
+    {
+        public IActionResult Index()
+        {
+            return View();
+        }
+    }
+}
+``` 
+There's only a few lines of code, but what's there is pretty powerful. First off, we inherit `Controller`, which gives us the ability to access the context of the request, information about the current user, and a way to leverage validation.  `Controller` also gives us a suite of helper methods that we can use to help generate responses.
+
+Secondly, we use one of the most common controller helpers in the action when we make a call to `View()`, which triggers the rendering of the view given the current state of our controller.
+
+Next, we have a return type of `IActionResult` giving us a flexible way to generate all different types of HTTP responses, from the typical `200 OK` to more complex responses like `302 Permanent Redirect` and a whole host of others.  These responses can be created using controller helpers, which we'll examine in greater detail later in this lesson.
+
+Finally, we have a method named `Index` and, along with our class named `PersonController`, users are now able to request the URL `person/`. This works because of the conventions and defaults of the framework and project template through a mechanism called routing.
+
+> Note that you can actually visit the `PersonController` if you run your app at this time and navigate to `~/Person` in the browser, but you will see an error because we haven't yet added a [view](views.md). You can, however, create a breakpoint in your controller on the `return View();` statement, which would allow you see execution flow through to that point.
```

Comment:
> navigate to `~/Person`

Should this be just "`/Person`"? I'm not sure readers will know what is the `~` supposed to represent.